README.md

```diff
@@ -57,9 +57,25 @@ via [npm (node package manager)](https://github.com/npm/npm)
 
     $ npm install sinon-test
 
-or just add it as a `<script src="dist/sinon-test.js"></script>`
-tag to the html where you write your tests. A pre-built browser
-version can be found in the NPM package under `dist/sinon-test.js`.
+or just add `dist/sinon-test.js` (or for ES6 Modules,
+`dist/sinon-test-es.js`) to where you write your tests.
+
+## Browser usage
+
+In place of the `require` statements indicated below, in the
```

Comment:
I'd place this below the other section (and reference _above_). Something like

# Usage 

## Node and CommonJS build systems
..
## Direct browser usage
..

Reply:
Sounds good, though Usage would be at heading level 2 to follow the current pattern...